### DISCUSSION TITLE
Problem with accessing Azure Student Offer

### DISCUSSION THREAD

**Discussion by PriyanshiKantariya** [community/community#138105] [GitHub Education] [answered]
### Hi everyone,

I'm a 3rd Year student, Pursuing Computer Science and Engineering. I have recently joined Github education community. I'm trying to claim Azure Student pack to learn DevOps. After I successfully completed the verification, at last page I got message that I'm not eligible though I got the successful verification message in previous stage.

**Comment by davevad93** [comment, marked as answer]
Hi @PriyanshiKantariya ,

This is a known issue as seen here #135739, to solve it you have the following options:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).
